frontend/src/component/common/Table/cells/FeatureOverviewCell/FeatureOverviewCell.tsx

```diff
@@ -207,7 +208,14 @@ const PrimaryFeatureInfo: FC<{
 
     const TypeIcon = () => (
         <HtmlTooltip arrow title={title} describeChild>
-            <IconComponent sx={(theme) => ({ fontSize: theme.spacing(2) })} />
+            <IconComponent
+                data-testid='feature-type-icon'
+                sx={(theme) => ({
+                    cursor: 'pointer',
+                    fontSize: theme.spacing(2),
+                })}
+                onClick={() => onTypeClick(type)}
```

Comment:
making feature type icon clickable

Reply:
Is it a button? 😉 Also, make sure the button has a label that says "add release flags to filter" or something, so it's clear what it does.

Reply:
will address in the next PR